flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/utils/SqlParserHelper.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.flink.table.catalog.hive.HiveTestUtils;
 import org.apache.flink.table.delegation.Parser;
 
-/** An utility class that provides pre-prepared tables and sql parser. */
+/** A utility class that provides pre-prepared tables and sql parser. */
```

Comment:
As as i know it should be `An` since it stays before the word starting with a vowel letter....
Or did I miss anything?

Reply:
"utility" begins with a vowel, it does not pronounce a vowel, but a consonant. Here is a funny website [a-or-an.com](https://www.a-or-an.com/a_an/utility), If you are not sure whether to use a or an, you can refer to it .